lib/cisco_node_utils/ace.rb

```diff
@@ -96,6 +99,60 @@ def ace_get
       regexp.match(str)
     end
 
+    # icmp ace getter
+    def icmp_ace_get(str)
+      # rubocop:disable Metrics/LineLength
```

Comment:
Do we need this ?

Reply:
Sorry, I am not following the comment. Are you asking if this method is needed? The answer is yes.

Reply:
If you are asking if the disable line length is needed, then yes it is required because of the regex line length more than 80 chars (just like the ace_get method where the original regex pattern is defined.